app/src/test/java/io/halkyon/CredentialsPageTest.java

```diff
@@ -15,9 +15,12 @@
 public class CredentialsPageTest extends BaseTest {
 
     @Test
-    public void testCreateNewCredential() {
+    public void testCreateNewBasicCredential() {
```

Comment:
Rename to testCreateUserPasswordCredential? it's more meaningful

Reply:
Fixed with: cfc1f25